EssentialsDiscordLink/src/main/java/net/essentialsx/discordlink/listeners/LinkBukkitListener.java

```diff
@@ -27,6 +32,20 @@ public LinkBukkitListener(EssentialsDiscordLink ess) {
         this.ess = ess;
     }
 
+    private void updateName(IUser user, InteractionMember member) {
+        switch (ess.getSettings().getNameSyncDirection()) {
+            case DISCORD_TO_MC:
+                // TODO: Remove this log
+                ess.getLogger().info(String.format("Setting username of %s to %s", user.getName(), member.getEffectiveName()));
+                // TODO: use a better way to change the display name
+                ess.getServer().dispatchCommand(Bukkit.getConsoleSender(), String.format("nickname %s %s", user.getName(), member.getEffectiveName()));
```

Comment:
Use the same code used in `/nick`: https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/main/java/com/earth2me/essentials/commands/Commandnick.java#L110-L117